cves/kernel/CVE-2019-19057.yml

```diff
@@ -450,7 +460,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The coding mistake that was made was simply forgetting to free two data structures after
+    a function failed. It appears to have been a lapse where the developer just never
+    remembered they needed to do that. 
+    
```

Comment:
Consider including a statement about how the fix for this vulnerability relates to the CWE, if it does

Reply:
Also look at the 2016 CVE mistakes section for the same suggestion